From 281ac54f21e2ddf91b213c3e1c4c419bd9d94ee8 Mon Sep 17 00:00:00 2001
From: Paul Eggert <eggert@cs.ucla.edu>
Date: Tue, 29 Jan 2013 13:46:48 -0800
Subject: [PATCH 1/4] autoscan: port to perl 5.17

* bin/autoscan.in (scan_sh_file): Escape '{'.  This avoids a
feature that is deprecated in Perl 5.17.  Reported by Ray Lauff in
<http://lists.gnu.org/archive/html/bug-autoconf/2013-01/msg00059.html>.
---
 bin/autoscan.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/bin/autoscan.in b/bin/autoscan.in
index a67c48d0..1f9809cc 100644
--- a/bin/autoscan.in
+++ b/bin/autoscan.in
@@ -358,7 +358,7 @@ sub scan_sh_file ($)
     {
       # Strip out comments and variable references.
       s/#.*//;
-      s/\${[^\}]*}//g;
+      s/\$\{[^\}]*}//g;
       s/@[^@]*@//g;
 
       # Tokens in the code.
-- 
2.13.0

